app/api/helpers/mail.py

```diff
@@ -369,12 +369,9 @@ def send_order_cancel_email(order):
         subject=MAILS[TICKET_CANCELLED]['subject'].format(
             event_name=order.event.name,
             invoice_id=order.invoice_number,
-            frontend_url=get_settings()['frontend_url']
```

Comment:
Why

Reply:
How is this field utilized in the subject, I didn't see it used anywhere in the text

Reply:
I do

Reply:
I got it!